src/Akka.HealthCheck.Persistence.Tests/TestConfig.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Akka.Util.Internal;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Akka.HealthCheck.Persistence.Tests
+{
+    public static class TestConfig
+    {
+
+        private static AtomicCounter counter = new AtomicCounter(0);
+
+        public static string config = @"akka.persistence {
```

Comment:
Turn this into a method that returns a `string` called `ValidConfig()` and have it take an optional integer argument, for instance:

```
public static string ValidConfig(int? dbId = null)
```

If `dbId` is `null`, then use the `counter` variable. That way the actual test itself can override the database ID if there's ever a need for it.